validators/crypto_addresses/eth_address.py

```diff
@@ -0,0 +1,57 @@
+"""ETH Address."""
+# -*- coding: utf-8 -*-
+
+# external
+import re
+
+from eth_hash.auto import keccak
+
+# local
+from validators.utils import validator
```

Comment:
@msamsami, I meant the change to be as follows:

```py
# standard
import re

# external
from eth_hash.auto import keccak

# local
from validators.utils import validator
```

As `re` is from the _standard_ library, `eth_hash.auto.keccak` is an _external_ dependency and `validators.utils.validator` implies a _local_ coupling.